.env

```
@@ -0,0 +1,45 @@
+# ===================================
+
+# Backend API Settings
+# -------------------
+REACT_APP_API_BASE_URL=https://ftapp.lfcww.org/api
+REACT_APP_LIVESTREAM_ENDPOINT=/Dashboard/togglelivestreamstate
+REACT_APP_UPDATE_LIVESTREAM_ENDPOINT=/Dashboard/UpdateAppSetting
+REACT_APP_POLLING_INTERVAL=30000
+
+# Admin Settings
+# -------------
+REACT_APP_ADMIN_PASSWORD="CHANGE_ME_ADMIN_PASSWORD"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)